test/test_request_verifier.py

```diff
@@ -0,0 +1,123 @@
+import sys
+import time
+import unittest
+
+from freeclimb.utils.request_verifier import RequestVerifier
+
+class TestRequestVerifier(unittest.TestCase):
+    """RequestVerifier unit test stubs"""
+
+    def setUp(self):
+        self.model = RequestVerifier()
```

Comment:
see comment referring to naming conventions

Reply:
will edit this in latest commit